### ISSUE TITLE
Know JBang version in custom JBang scripts or commands

### ISSUE THREAD

**Issue by davsclaus** [jbangdev/jbang#1440]
**Is your feature request related to a problem? Please describe.**

To know the jbang version at runtime in custom jbang app (eg camel-jbang)

**Describe the solution you'd like**

I would like jbang to automatic set a JVM system property with the jbang version so I can find out without having to use any jbang API as I need this in a neutral way.

In my camel jbang commands then I only have dependency on picocli and therefore cannot use jbang JARs.

**Describe alternatives you've considered**

**Additional context**

**Comment by quintesse** [comment]
That's not a bad idea. 👍

**Comment by davsclaus** [comment]
Here is a sample what the Camel CLI we are working on
```
~/workspace ❯ camel get
  PID   NAME      CAMEL            PLATFORM  READY   STATE   AGE  TOTAL  FAILED  INFLIGHT  SINCE-LAST
 49448  bar.yaml  3.19.0-SNAPSHOT  JBang      1/1   Started   6s      3       0         0          0s
```

In the PLATFORM column I would like to say
```
JBang v0.97.0
```

**Comment by maxandersen** [comment]
This would probably better to be environment variables rather than system properties to be consistent when using native built apps. 

+1 on the feature.

**Comment by davsclaus** [comment]
There is a `version.txt` in the cache folder. 

~/.jbang/cache/version.txt

This file has 0.106.1 on my laptop.

But I just upgraded to 0.106.3 via skdman

~/workspace ❯ cat ~/.jbang/cache/version.txt
0.106.1%
~/workspace ❯ jbang -V
0.106.3

**Comment by quintesse** [comment]
Yes, that only gets updated once JBang checks for a new version (which you can force by running `jbang version -- check` IIRC)
